tests/coprocessor/test_select.rs

```diff
@@ -838,7 +126,15 @@ fn test_stream_batch_row_limit() {
         let engine = engine::new_local_engine(TEMP_DIR, ALL_CFS).unwrap();
         let mut cfg = Config::default();
         cfg.end_point_stream_batch_row_limit = stream_row_limit;
-        init_data_with_details(Context::new(), engine, &product, &data, true, cfg)
+        init_data_with_details(
+            Context::new(),
+            engine,
+            &product,
+            &data,
+            true,
+            &cfg,
+            &readpool::Config::default_for_test(),
```

Comment:
Why change to `readpool::Config::default_for_test`?

Reply:
Previously it is default_for_test as well, this PR just makes it configurable in the underlying function from arguments.